lib/galaxy/datatypes/binary.py

```diff
@@ -169,6 +169,11 @@ def _get_samtools_version( self ):
                 break
         return version
 
+    @staticmethod
+    def merge(split_files, output_file):
+        params = ["samtools", "merge", "-f", output_file, " ".join(split_files)]
```

Comment:
I assume split_files is a list, if yes then something like this should work:

params = ["samtools", "merge", "-f", output_file] + split_files

I would then set ``shell=False`` or remove the parameter and should be the default.